services/app-api/resolvers/indexSubmissions.ts

```diff
@@ -6,17 +6,22 @@ import {
 import { QueryResolvers } from '../gen/gqlServer'
 import { logError, logSuccess } from '../logger'
 import { isStoreError, Store } from '../postgres'
+import { setContextualAttributes, setErrorAttributes, setSuccessAttributes } from './attributeHelper'
 
 export function indexSubmissionsResolver(
     store: Store
 ): QueryResolvers['indexSubmissions'] {
     return async (_parent, _args, context) => {
+        const { user, span } = context
+        setContextualAttributes('indexSubmissions', user, span)
+        
         // This resolver is only callable by state users
         if (!isStateUser(context.user)) {
             logError(
```

Comment:
do we still need `logError` now?  Can we just remove logger or make logError be what calls `setErrorAttributes`

Reply:
I think logError/logSuccess will go away once we have instrumentation.  I guess it's a question for @macrael: does anything in any of our scripts depend on the error/success logs?  If not, we can remove them as we add instrumentation...

Reply:
let’s leave them in for now, we can remove them once we feel like we’ve gotten used to instrumentation. 

I don’t want to give up being able to look at the CloudWatch and see what’s up. If our otel logs are showing up there now, then we can probably ditch the logError stuff since we’ll still have something in cloudwatch if we look at it.